edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -34,11 +34,6 @@
       {% endblocktrans %}
     </p>
     <p>
-      {% blocktrans %}
-        You will be asked to verify your identity before you begin the exam. Make sure you have valid photo identification, such as a driver's license or passport, before you continue.
```

Comment:
Interesting. Removing this line was in the acceptance criteria:

> Once this verification step is added, we can remove the line of text in the Proctoring Exam Instructions that reads: "You will be asked to verify your identity before you begin the exam. Make sure you have valid photo identification, such as a driver's license or passport, before you continue." 
This existing step in the process will no longer need to happen within the exam workflow as it will be completed prior to start of the proctoring instructions step.

I'll add it back with the revised wording.